swift_kernel.py

```diff
@@ -490,6 +498,20 @@ def do_execute(self, code, silent, store_history=True,
                 'user_expressions': {}
             }
         elif isinstance(result, ExecutionResultError):
+            if not self.process.is_alive:
+                error_message = self._make_error_message(['Process killed'])
```

Comment:
Where will the error message show up? Will the users notice it so that they know to recompute the exiting cells?

Reply:
It shows up in the output for the cell that was executing when the process got killed:
![screenshot from 2019-02-22 11-54-47](https://user-images.githubusercontent.com/5945760/53267628-b15b7280-3698-11e9-844b-a12b05b8c220.png)

I think it's pretty noticeable because the user will be looking at the output of the cell that they are executing. Also, the UI shows an additional message that the kernel has been restarted.